docs/cn/build_and_run.md

```diff
@@ -30,16 +30,23 @@ docker build -t opencurvedocker/curve-base:build-debian9
 ### 在docker镜像中编译
 
 ```bash
-docker run -it opencurvedocker/curve-base:build-debian9 /bin/bash
-cd <workspace>
 git clone https://github.com/opencurve/curve.git 或者 git clone https://gitee.com/mirrors/curve.git
+cd curve
+# 如果你想在容器内完成编译+制作+上传镜像的操作，可以添加以下参数
+# -v /var/run/docker.sock:/var/run/docker.sock -v /root/.docker:/root/.docker
+docker run -v $(pwd):/curve -w /curve -v /root/.cache/bazel:/root/.cache/bazel  --privileged -it opencurvedocker/curve-base:build-debian9 bash
 # （中国大陆可选）将外部依赖替换为国内下载点或镜像仓库，可以加快编译速度： bash replace-curve-repo.sh
+
 # curve v2.0 之前
 bash mk-tar.sh （编译 curvebs 并打tar包）
 bash mk-deb.sh （编译 curvebs 并打debian包）
+
 # （当前）curve v2.0 及之后
-编译 curvebs: cd curve && make build stor=bs dep=1
-编译 curvefs: cd curve && make build stor=fs dep=1
+# 注意以下方式不会编译单元测试的代码
+# 编译 curvebs:
+make dep stor=bs && make build stor=bs only=src,tools
```

Comment:
Does nebd and nbd will compiled with this command?

Reply:
> Does nebd and nbd will compiled with this command?

fix